spec/factories/session.rb

```diff
@@ -83,19 +83,6 @@
       national_insurance { 112 }
     end
 
-    trait :with_other_income do
```

Comment:
is there a reason why you have deleted this trait?

Reply:
I had used it then removed it, then found it was not in use, the one place it was being used in a spec it wasn't actually required for the spec. So I figured it should be removed if it wasnt being used and it didnt warrant its own ticket or PR.